python/lib/dependabot/python/update_checker/pipenv_version_resolver.rb

```diff
@@ -88,7 +85,7 @@ def fetch_latest_resolvable_version_string(requirement:)
                 # Whilst calling `lock` avoids doing an install as part of the
                 # pipenv flow, an install is still done by pip-tools in order
                 # to resolve the dependencies. That means this is slow.
-                run_pipenv_command("pyenv exec pipenv lock")
+                run_pipenv_command("pyenv exec pipenv lock --dev")
```

Comment:
Also include dev dependencies: https://pipenv-fork.readthedocs.io/en/latest/advanced.html#generating-a-requirements-txt

> As with other commands, passing `--dev` will include both the default and development dependencies

Reply:
Did this behaviour change in the new pipenv version? Does this also apply to the `Pipfile.lock`? Docks only referenced generating `requirements.txt` files with the `-r` option (`pipenv lock -r --dev`)

Reply:
Aye, yes this may not be needed in that case!